tests/demo_tests/test_ip_v4v6_raw_capture.py

```diff
@@ -0,0 +1,196 @@
+import pytest
+import snappi
+
+
+@pytest.fixture
+def app_intf(utils):
+    api = snappi.api(host=utils.settings.api_server, ext=utils.settings.ext)
+    yield api
+    if getattr(api, 'assistant', None) is not None:
+        api.assistant.Session.remove()
+
+
+def test_ip_v4v6_raw_capture(app_intf, utils):
+    """
+    Test to demonstrate ipv4 and ipv6 device and raw type traffic
+    configuration, transmit, statistics and capture.
+    """
+    config = app_intf.config()
+
+    size = 256
+    packets = 1000
+    src = '00:10:10:20:20:10'
```

Comment:
for consistency, use `src_mac` and `dst_mac`

Reply:
addressed